Automattic/vip-cli
# Add sync preview

This adds sync preview to the `vip sync` command.

### How to test?

1. clone the branch
2. run `npm link`
3. run `vip sync --app=XXX` where `XXX` is the id of an app
4. select the branch
5. verify if everything looks good
6. type N to abort sync

### Screenshot
<img width="402" alt="screen shot 2018-10-24 at 9 32 02 am" src="https://user-images.githubusercontent.com/12542046/47417668-70963000-d770-11e8-96cd-ae0431097dd5.png">

fixes #164
